test/service_consumers/pact_helper.rb

```diff
@@ -10,13 +10,15 @@
   config.include GdsApi::TestHelpers::ContentStore
 end
 
+WebMock.allow_net_connect!
+
 class ProxyApp
   def initialize(real_provider_app)
     @real_provider_app = real_provider_app
   end
 
   def call(env)
-    env["HTTP_HOST"] = "localhost:3003"
+    env["HTTP_HOST"] = "localhost"
     response = @real_provider_app.call(env)
     response
```

Comment:
```suggestion
    @real_provider_app.call(env)
```

Not related to this PR, but I did notice this could be a bit tidier.